management/http_recordings_test.go

```diff
@@ -8,89 +8,97 @@ import (
 	"testing"
 	"time"
 
-	"github.com/dnaeon/go-vcr/v2/cassette"
-	"github.com/dnaeon/go-vcr/v2/recorder"
 	"github.com/stretchr/testify/require"
+	"gopkg.in/dnaeon/go-vcr.v3/cassette"
+	"gopkg.in/dnaeon/go-vcr.v3/recorder"
 
 	"github.com/auth0/go-auth0"
 )
 
-func setupHTTPRecordings(t *testing.T) {
+const (
+	recordingsDIR    = "./../test/data/recordings/"
+	recordingsDomain = "go-auth0-dev.eu.auth0.com"
+)
+
+func configureHTTPTestRecordings(t *testing.T) {
 	t.Helper()
 
 	if !httpRecordingsEnabled {
 		return
 	}
 
-	const recordingsDIR = "testdata/recordings/"
-
-	initialTransport := m.http.Transport
+	initialTransport := api.http.Transport
 
-	vcrTransport, err := recorder.NewAsMode(
-		recordingsDIR+t.Name(),
-		recorder.ModeReplaying,
-		m.http.Transport,
+	recorderTransport, err := recorder.NewWithOptions(
+		&recorder.Options{
+			CassetteName:       recordingsDIR + t.Name(),
+			Mode:               recorder.ModeRecordOnce,
+			RealTransport:      api.http.Transport,
+			SkipRequestLatency: true,
```

Comment:
We skip request latency, but we still save it the first time. This will enable us in the future to introduce test that depend on latency specific behavior.